cypress/integration/interpretations.cy.js

```diff
@@ -82,6 +83,29 @@ describe('interpretations', () => {
         expectInterpretationFormToBeVisible()
     })
 
+    it('the interpretations panel can be toggled by clicking the option in the view menu', () => {
+        // Hiding (the interpretations panel is open at the start of the test)
```

Comment:
Isn't `clickMenubarViewButton` showing the panel here?
I'm a bit confused by this comment.